packages/tools/viewer-alpha/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "@tools/viewer-alpha",
```

Comment:
@RaananW we had to give this a unique name so it doesn't conflict with the existing `@tools/viewer` in the context of this mono repo and npm workspace. Does this seem like the right thing to do? I assume for the packages that we actually publish we can still use `@babylonjs/viewer` and publish with the `-alpha` version suffix and `@preview` tag as planned?